panel/browser-panel.cpp

```diff
@@ -270,7 +271,6 @@ void QCefWidgetInternal::resizeEvent(QResizeEvent *event)
 
 void QCefWidgetInternal::Resize()
 {
-#ifdef _WIN32
```

Comment:
If you're removing the `#ifdef _WIN32` here for `SUPPORTS_FRACTIONAL_SCALING`, doesn't that mean the instance on line 233/235 should also be moved/removed?

Reply:
Fixed.